subsys/bluetooth/host/audio/media_proxy.h

```diff
@@ -521,24 +612,41 @@ struct media_proxy_ctrl_cbs {
 	 *                     or errno on negative value.
 	 * @param opcodes      The supported command opcodes (bitmap)
 	 */
-	void (*commands_supported)(struct media_player *player, int err, uint32_t opcodes);
+	void (*commands_supported_recv)(struct media_player *player, int err, uint32_t opcodes);
 
 	/**
-	 * @brief Search callback
+	 * @brief Search send callback
 	 *
-	 * Called when a search operation has been set, to give the result
-	 * of the operation
+	 * Called when a search has been sent
 	 * See also media_proxy_ctrl_search_send()
 	 *
 	 * @param player        Media player instance pointer
 	 * @param err           Error value. 0 on success, GATT error on positive value
 	 *                      or errno on negative value.
 	 * @param result_code   The result of the operation
 	 */
-	void (*search)(struct media_player *player, int err, uint8_t result_code);
+	void (*search_send)(struct media_player *player, int err);
 
 	/**
-	 * @brief Search Results Object ID callback
+	 * @brief Search result code receive callback
+	 *
+	 * Called when a search result code has been received
+	 * See also media_proxy_ctrl_search_send()
+	 *
+	 * The search result code tells whether the search was successful or not.
+	 * For a successful search, the actual results of the search (i.e. what was found
+	 * as a result of the search)can be accessed using the Search Results Object ID.
+	 * The Search Results Object ID has a separate callback - search_results_id_recv().
+	 *
+	 * @param player        Media player instance pointer
+	 * @param err           Error value. 0 on success, GATT error on positive value
+	 *                      or errno on negative value.
+	 * @param result_code   Search result code
+	 */
+	void (*search_recv)(struct media_player *player, int err, uint8_t result_code);
```

Comment:
Not sure if you have it on a TODO somewhere already, but please remember adding values for the `result_code` values in `media_proxy.h`. It currently seems like the implementation uses the `BT_MCS_OPC_NTF_` values.

Reply:
Will make a task for that.  It does require a bit more of changes, as some of the BT_MCS se are also CONFIG values.

Reply:
There are result code values in the `media_proxy.h`, though:
`/**
 * @brief Search operation result codes
 */
#define MEDIA_PROXY_SEARCH_SUCCESS  0x01
#define MEDIA_PROXY_SEARCH_FAILURE  0x02`

Reply:
It seems to a bit of a mix. Some places use `MEDIA_PROXY_SEARCH_` but e.g. `media_proxy_pl_search_cb` uses the `BT_MCS_SCP_NTF_` values. 

I also see that my initial comment was incorrect, as the values I referred to was used for the `mpl_cmd_ntf`notifications, but are defined in `mcs.h`.